docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -491,6 +491,7 @@ Below is the list of the supported config properties.
 | `keymap.newTerminalTab`       | `` Control+Shift+` `` on macOS<br/>`` Ctrl+Shift+` `` on Windows/Linux                                               | Shortcut to open a new terminal tab.                                                                       |
 | `keymap.terminalCopy`         | `Command+C` on macOS<br/>`Ctrl+Shift+C` on Windows/Linux                                                             | Shortcut to copy text in the terminal.                                                                     |
 | `keymap.terminalPaste`        | `Command+V` on macOS<br/>`Ctrl+Shift+V` on Windows/Linux                                                             | Shortcut to paste text in the terminal.                                                                    |
+| `keymap.terminalSearch`       | `Command+F` on macOS<br/>`Ctrl+F` on Windows/Linux                                                                   | Shortcut to open a search field in the terminal.                                                           |
```

Comment:
FYI this needs to be added to the list of shortcuts in the docs too https://github.com/gravitational/teleport/blob/a930d3eb3cbc924b12d279ac15b0721e855bd613/docs/pages/connect-your-client/teleport-connect.mdx.
But you can do this in a separate PR.